Assets/Mapzen/TileBounds.cs

```diff
@@ -0,0 +1,56 @@
+using System;
+using System.Collections.Generic;
+
+namespace Mapzen
+{
+    [System.Serializable]
+    public struct TileArea
+    {
+        public LngLat min;
+        public LngLat max;
+        public int zoom;
+
+        public TileArea(LngLat min, LngLat max, int zoom)
+        {
+            this.min = min;
+            this.max = max;
+            this.zoom = zoom;
+        }
+    }
+
+    public class TileBounds
+    {
+        public readonly TileAddress min;
+        public readonly TileAddress max;
+
+        private TileArea area;
+
+        public TileBounds(TileArea area)
+        {
+            this.area = area;
+
+            min = TileAddress.FromLngLat(area.min, area.zoom);
```

Comment:
Oh that's good to know, I half-expected that Unity would run debug builds in the Editor but I guess not. 

It also wouldn't be too hard to just accept reversed min/max pairs and find a suitable bounding box.